spec/chewy/fields/base_spec.rb

```diff
@@ -40,9 +44,25 @@
     end
 
     context 'parent objects' do
-      let!(:country) { described_class.new(:name, value: ->(country, crutches) { country.cities.map { |city| double(districts: city.districts, name: crutches.city_name) } }) }
-      let!(:city) { described_class.new(:name, value: ->(city, country, crutches) { city.districts.map { |district| [district, country.name, crutches.suffix] } }) }
-      let!(:district) { described_class.new(:name, value: ->(district, city, country, crutches) { [district, city.name, country.name, crutches] }) }
+      let!(:country) do
+        described_class.new(:name, value: lambda { |country, crutches|
+                                            country.cities.map do |city|
+                                              double(districts: city.districts, name: crutches.city_name)
+                                            end
+                                          })
+      end
+      let!(:city) do
+        described_class.new(:name, value: lambda { |city, country, crutches|
+                                            city.districts.map do |district|
+                                              [district, country.name, crutches.suffix]
+                                            end
+                                          })
+      end
+      let!(:district) do
+        described_class.new(:name, value: lambda { |district, city, country, crutches|
+                                            [district, city.name, country.name, crutches]
+                                          })
+      end
```

Comment:
Maybe it would be even more readable to have lambda values defined in other let's